docs/source/topics/build-examples.rst

```diff
@@ -26,24 +26,53 @@ basic usage scenarios of |short_name| with SYCL*. Go to
 
 .. note::
 
-   All content starting with a # below is considered a comment and
+   All content below that starts with ``#`` is considered a comment and
    should not be run with the code.
 
 1. Set up the |short_name| environment:
 
-  **Linux\*:**
+  .. tabs::
 
-    .. substitution-prompt:: bash
+    .. group-tab:: Linux\*
```

Comment:
Technically we also have MacOS supported in opensource form. A far this documentation is also covering opensource cases - might be this should be reflected. 
This is comment for future discussion - i'm not expecting to have any fixes in this PR

Reply:
ok, noted
I'd need someone to provide me with this content (commands, instructions. etc.)